analysis/org.eclipse.tracecompass.analysis.counters.core/src/org/eclipse/tracecompass/analysis/counters/core/CounterStateProvider.java

```diff
@@ -147,8 +147,29 @@ protected void eventHandle(@NonNull ITmfEvent event) {
      *            Grouped counter aspect
      * @param rootQuark
      *            Key to a relative root of the state system
+     * @deprecated use
+     *             {@link #handleGroupedCounterAspect(ITmfEvent, ITmfStateSystemBuilder, int, ITfmCounterAspect)}
      */
+    @Deprecated
     protected void handleGroupedCounterAspect(ITmfEvent event, ITmfStateSystemBuilder ss, CounterAspect aspect, int rootQuark) {
+        handleGroupedCounterAspect(event, ss, rootQuark, aspect);
+    }
+
+    /**
+     * Add the field value of a grouped counter aspect to the state system
+     * (override in specific implementations)
+     *
+     * @param event
+     *            Event to process
+     * @param ss
+     *            State system object to fill
+     * @param rootQuark
+     *            Key to a relative root of the state system
+     * @param aspect
+     *            Grouped counter aspect
+     * @since 2.2
```

Comment:
API change requires bump to 2.3.0 in MANIFEST.MF and since tag changed to 2.3 here.